packages/lesswrong/lib/collections/users/fragments.ts

```diff
@@ -81,6 +81,9 @@ registerFragment(`
     paymentEmail
     paymentInfo
     goodHeartTokens
+    bannedFromPosting
+    bannedFromCommenting
+    bannedFromPMing
   }
```

Comment:
What about banning from voting? Not sure whether that should be included or not. I do think banned users are people whose judgment we might dislike and who might do more retalitory stuff

Reply:
I do think we should have banning from voting but voting was a bit of a weirder system to interact with, so was kinda punting it to the future.

Reply:
(part of my decisionmaking here is "this isn't the top priority so I wanted to do stuff that was a simple improvement that didn't require much effort to reason about.")